rero_ils/modules/acq_invoices/api.py

```diff
@@ -145,6 +146,12 @@ def vendor_pid(self):
         """Shortcut for acquisition order vendor pid."""
         return extracted_data_from_ref(self.get('vendor'))
 
+    @property
+    def is_active(self):
```

Comment:
If it is always `True` is it really useful?

Reply:
Yes, and there is a comment `# TODO: implement this when introducing the invoicing module` for later implementation of the invoicing system